src/main/scala/viper/silver/plugin/standard/inline/InlineErrorChecker.scala

```diff
@@ -93,9 +93,9 @@ trait InlineErrorChecker {
   private[this] def prettyPrint(preds: Set[Predicate], errorReason: String): Unit = {
     val predIds = preds.map(_.name).mkString(", ")
     if (preds.size > 1) {
-      println(s"[$predIds] are $errorReason predicates and will not be inlined.")
+      println(s"${Console.RED} [$predIds] are $errorReason predicates and will not be inlined.")
```

Comment:
is there a `Console.RAINBOW`

Reply:
I'm intrigued
![image](https://user-images.githubusercontent.com/24359440/102815539-57559200-4381-11eb-867e-2ffd8390088e.png)

Reply:
Should this be yellow instead of red since it's just a warning? The [info] msgs are yellow too

Reply:
nvm you've already merged, doesn't matter lol

Reply:
make every other word a different colour imo :rainbow_flag:

Reply:
Set to `Yellow` via push to master (forgive me)